### PR TITLE
Fix/oo customized queries respect order

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: app/scripts/reports/detail/reportsdetail.controller.js [hunk 1]
```diff
@@ -111,20 +101,52 @@ angular.module('openolitor-admin')
...
           }
         });
       };
 
+      $scope.getValue = function(col, dataEntry) {
+        return $scope.renderedInput(col,dataEntry[col]);
+      }
+
+      $scope.replaceJsonNotAllowedCharacters = function (jsonString) {
+         return jsonString.replace('\n',' ')
```
Comment (AlwinEgger) [inline app/scripts/reports/detail/reportsdetail.controller.js]: Could be repalced with nicer regex

File: app/scripts/reports/detail/reportsdetail.html [hunk 2]
```diff
@@ -92,8 +92,8 @@ <h3 class="panel-title"><span class="navbar-brand" translate>Resultate</span>
             <tr>
                 <td ng-repeat="col in $columns" ng-bind-html="renderedTitles(col.field)"></td>
             </tr>
-            <tr ng-repeat="dataEntry in $data | filter: search.query.$">
-              <td ng-repeat="col in $columns" ng-bind-html="col.getValue(this, dataEntry)"></td>
+            <tr ng-repeat="dataEntry in result.entries | filter: search.query.$">
+              <td ng-repeat="col in $columns" ng-bind-html="getValue(col.field,dataEntry, result.entries)"></td>
```
Comment (AlwinEgger) [inline app/scripts/reports/detail/reportsdetail.html]: getValue does not have a third param, right?

**Review by AlwinEgger — CHANGES_REQUESTED** [review]

File: app/scripts/reports/detail/reportsdetail.controller.js (hunk 1 above)
Comment (mcmpp) [reply app/scripts/reports/detail/reportsdetail.controller.js]: done

File: app/scripts/reports/detail/reportsdetail.html (hunk 2 above)
Comment (mcmpp) [reply app/scripts/reports/detail/reportsdetail.html]: done

**Review by AlwinEgger — APPROVED** [review]

**Pull request merged**